test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -55,36 +55,37 @@ string ProtoConverter::createAlphaNum(string const& _strBytes) const
 
 bool ProtoConverter::isCaseLiteralUnique(Literal const& _x)
 {
-	std::string tmp;
+	dev::u256 mpCaseLiteralValue;
 	bool isUnique = false;
 	bool isEmptyString = false;
+
 	switch (_x.literal_oneof_case())
 	{
 	case Literal::kIntval:
-		tmp = std::to_string(_x.intval());
+		mpCaseLiteralValue = dev::u256(std::to_string(_x.intval()));
```

Comment:
Is it necessary to convert to string first?

Reply:
No, I'll change it.